src/main/java/reposense/git/GitUtil.java

```diff
@@ -80,14 +86,54 @@ static String convertToGitFormatsArgs(List<Format> formats) {
     /**
      * Returns the {@code String} command to specify the globs to exclude for `git log` command.
      */
-    static String convertToGitExcludeGlobArgs(List<String> ignoreGlobList) {
+    public static String convertToGitExcludeGlobArgs(File root, List<String> ignoreGlobList) {
         StringBuilder gitExcludeGlobArgsBuilder = new StringBuilder();
         final String cmdFormat = " " + addQuote(":(exclude)%s");
         ignoreGlobList.stream()
-                .filter(item -> !item.isEmpty())
+                .filter(item -> !item.isEmpty() && isValidPath(root, item))
```

Comment:
It seems that my explanation in this https://github.com/reposense/RepoSense/pull/592#discussion_r264026681 is not clear enough.

I mean that this filter will automatically exclude the invalid path which may not throw exceptions. So, when ignoring these kinds of invalid paths, it is better to give a log message to warn the user that, there is an invalid path in the ignoreGlobList and this path is skipped. Having this message, the user will realize that there may be a mistake in the csv file.